google/cloud/ndb/tasklets.py

```diff
@@ -330,7 +330,7 @@ def done_callback(yielded):
             #
             # It was tempting to call `_advance_tasklet` (`_help_tasklet_along`
             # in Legacy) directly. Doing so, it has been found, can lead to
-            # exceeding the maximum recursion depth. Queing it up to run on the
+            # exceeding the maximum recursion depth. Queuing it up to run on the
```

Comment:
The correction made the line length go over 79 characters, which is not accepted by the lint checker. Please move the last word here to the following line.

Reply:
Done in e239dc9e348681c4bf327fe75341932873014bfc